README.md

```diff
@@ -1 +1,72 @@
-# Hello-Microverse-Project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This project is a simple exercise on creating Github repository, branches and github flows.
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters
+
+## Live Demo (if available)
+
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job documenting your code! 💯  However, I'm sorry that the last reviewer didn't point out this, but there's an updated version of the [Microverse's template](https://github.com/microverseinc/readme-template). Kindly use it to document your code. Remember that having professional code documentation will make your projects more attractive to people that want to collaborate on them and also will provide better guidance on how to interact with them. Please consider using the template and customizing it with the project information! 🤓 🛠️